packages/client/src/menu/guilds.ts

```diff
@@ -333,6 +340,21 @@ export default class Guilds extends Menu {
         this.currentSidebar = menu;
     }
 
+    private appendChatLog(log: ChatLog):void {
```

Comment:
There is already a function for this - see `handleChat`. You could probably cast the log as a GuildPacket.

Reply:
nvm